packages/react-instantsearch-core/src/widgets/InstantSearch.tsx

```diff
@@ -5,7 +5,11 @@ import createInstantSearchManager from '../core/createInstantSearchManager';
 import { InstantSearchProvider, InstantSearchContext } from '../core/context';
 import { Store } from '../core/createStore';
 import { PlainSearchParameters, SearchParameters } from 'algoliasearch-helper';
-import { MultiResponse } from 'algoliasearch';
+import { SearchResponse } from '@algolia/client-search';
```

Comment:
Ups - is this code work on the v3 too?

Reply:
I changed it to be closer to what we do in InstantSearch.js, same for the test with the client v3 on the CI.